### DISCUSSION TITLE
New-Point Blank patch+workaround

### DISCUSSION THREAD

**Discussion by DatDerpyDude81** [3kh0/ext-remover#673] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
So basically, in my school the link for securly in the new point blank section is bugged. When entering this link, the page shows "blocked by chrome." Additionally, reloading just sends you to chrome-extension://invalid, which is also blocked.

I'm not sure if this issue has happened to others, but apprently the extensionid is incorrect(securly). If you have this issue, try going to the following link:

[chrome-extension://joflmkccibkooplaeoinecjbmdebglab/blocked.html?category=PGEgaHJlZj0iYWJvdXQ6YmxhbmsiIHRhcmdldD0iX2JsYW5rIiByZWw9Im9wZW5lciIgc3R5bGU9IiBiYWNrZ3JvdW5kOiB3aGl0ZTsgY29sb3I6IGJsYWNrOyBkaXNwbGF5OiBibG9jazsgaGVpZ2h0OiAzODBweDsgdHJhbnNmb3JtOiBzY2FsZSgyLjUpOyBkaXNwbGF5OiBmbGV4OyBqdXN0aWZ5LWNvbnRlbnQ6IGNlbnRlcjsgYWxpZ24taXRlbXM6IGNlbnRlcjsgIj48ZGl2PkNsaWNrIE1lPC9kaXY%2BIDwvYT4%3D](url)

(if this is a github 404 link just copy and paste the url into your browser)

**Comment by Aka-but-nice** [comment]
It's because your school switched to another Securly, which is not on the Chrome Webstore.

**Comment by synapt0** [comment]
The point blank bookmarklet still doesn't work for me.
